docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+
+## Create a Fabric workspace
+
+1. Sign in to the Fabric app using your [Fabric free account](https://www.microsoft.com/microsoft-fabric/getting-started) or [Microsoft Power BI](https://app.powerbi.com) account.
+2. You can create a new workspace for your FinOps resources or choose to use an existing workspace. To create a workspace, see [Create a workspace](/fabric/get-started/create-workspaces).
+
+If you already have a [Microsoft Fabric workspace](/fabric/get-started/workspaces), you can skip creating a new workspace, unless you want to keep your FinOps resources separate from others.
+
+If you didn't set up Microsoft Fabric, see [Enable Microsoft Fabric for your organization](/fabric/admin/fabric-switch).
+
+If you’re new to Microsoft Fabric or don’t have Fabric capacity enabled, see [Fabric trial](/fabric/get-started/fabric-trial). If Fabric trials are disabled, you might need to [create a new Fabric capacity](/fabric/admin/capacity-settings?tabs=fabric-capacity#create-a-new-capacity).
+
+Nontrial Fabric capacity incurs charges for your organization. See [Microsoft Fabric pricing](https://azure.microsoft.com/pricing/details/microsoft-fabric/).
+
+Here's an example screenshot showing a new Fabric workspace getting created.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/fabric-create-workspace.png" border="true" alt-text="Screenshot showing a workspace getting created." lightbox="./media/create-fabric-workspace-finops/fabric-create-workspace.png" :::
+
+## Create a lakehouse
+
+You can either create a new lakehouse or use an existing one. Lakehouses are interoperable so we would generally recommend using a single lakehouse for all your FinOps datasets. A single lakehouse keeps related information together to share data models, pipelines, and security measures.
+
+To create a new lakehouse in the Fabric workspace of your choice:
+
+1. Select **New** > **More options**.  
+    :::image type="content" source="./media/create-fabric-workspace-finops/fabric-new-more-options.png" border="true" alt-text="Screenshot showing navigation to More options." lightbox="./media/create-fabric-workspace-finops/fabric-new-more-options.png" :::
+3. Select **Lakehouse**  
+    :::image type="content" source="./media/create-fabric-workspace-finops/fabric-new-lakehouse.png" border="true" alt-text="Screenshot showing the Lakehouse option." lightbox="./media/create-fabric-workspace-finops/fabric-new-lakehouse.png" :::
+
+For more information, see [Create a lakehouse](/fabric/data-engineering/create-lakehouse).
+
+>[!NOTE]
+>Microsoft Fabric [lakehouses](/fabric/data-engineering/lakehouse-overview) are data architectures that allow organizations to store and manage structured and unstructured data in a single location. They use  various tools and frameworks to process and analyze that data. These tools and frameworks can include SQL-based queries and analytics, machine learning, and other advanced analytics techniques.
```

Comment:
nit: Should this be the intro paragraph for this section? I'm fine either way.

Reply:
Moved to section intro as suggested.